src/index.html

```diff
@@ -10,7 +10,28 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class ="search-form search-form_big" data-qa="big">
+      <label class ="search-form__label search-form__label_big">
+        <input
```

Comment:
It's recommended to add `id` and `name` attributes to input to avoid weird behavior and make sure the data is always sent correctly